k-distribution/tests/regression-new/issue-2273/tests/kast-in1.kast.out

```diff
@@ -0,0 +1,5 @@
+[Error] Inner Parser: Could not find start symbol: Foo
+	Source(tests/kast-in1.kast)
+	Location(1,1,1,2)
```

Comment:
Only complaint is that this error message maybe doesn't make sense for a source location. Could we somehow say "Source location: CLI --sort flag" instead or something?

Reply:
I had to carry a new parameter on all locations where we call the parser.
The code doesn't look very pretty though.
If you think it's worth it, add automerge. Otherwise, I can revert the last commit and then merge.